apps/demo/src/app/shared/components/node-editor/node-editor.component.html

```diff
@@ -3,6 +3,7 @@
   [placeholder]="placeholder()"
   (keyup.enter)="save.emit(value)"
   (keyup.escape)="cancel.emit()"
+  (keydown)="keydownHandler($event)"
```

Comment:
This is intentional.  The issue is that the tree control this component is in is handling the keys and we don't want them to be handled while we are editing. Since the tree is a 3rd party component, we prevent the event propagation here.